homeassistant/components/sensor/uptime.py

```diff
@@ -18,33 +18,30 @@
 
 DEFAULT_NAME = 'Uptime'
 
-ICON = 'mdi:clock'
+ICON = 'mdi:timer'
+
+DEVICE_CLASS = 'datetime'
```

Comment:
Allowed device classes are defined in sensor component. I also think that just datetime might not be a good descriptor, maybe something as period? I think we should open an issue in the architecture repo about this first.